src/graphics/primitives/tiles/atlas.rs

```diff
@@ -244,9 +244,26 @@ impl Atlas {
     }
 
     pub fn calc_quad(&self, tile: i16, x: usize, y: usize) -> Quad {
+        let tile_u32 = if tile < 0 { 0 } else { tile as u32 };
+
+        let atlas_tile_position = if tile_u32 < AUTOTILE_ID_AMOUNT {
+            egui::pos2(0., 0.)
+        } else {
+            egui::pos2(
+                (((tile_u32 - AUTOTILE_ID_AMOUNT) % AUTOTILE_FRAME_COLS) * TILE_SIZE) as f32,
```

Comment:
This this account for atlas wrapping?

Reply:
No but neither does tilemap.wgsl (or so I assume). I guess I can quickly fix this.

Reply:
It doesn't, I could never figure it out, and I couldn't explain the problem well enough to anyone else to get them to help either